### PR TITLE
Add gamma module

### PR DESCRIPTION
**Pull request by Mridul97**
Issue #368 

Changes : A module is added which applies gamma correction on an image and takes gamma as an input.

### CODE REVIEW

**Comment by Mridul97** [comment]
This is a screenshot of the output with gamma = 0.2 :

ss_13

File: src/modules/Gamma/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Gamma",
+    "description": "Apply gamma correction on the image",
+    "inputs": {
+        "adjustment": {
+            "type": "float",
+            "desc": "gamma correction for the new image",
```
Comment (tech4GT) [inline src/modules/Gamma/info.json]: Just to be extra clear let's also mention this is inverse of the actual gamma.

Comment (Mridul97) [reply src/modules/Gamma/info.json]: I will change the module name to Gamma Correction and also add a link to Wikipedia's page on Gamma to the description. Thanks!

I didn't quite get 'inverse of the actual gamma'. I think it is gamma, how is this inverse of the actual gamma?

Comment (tech4GT) [reply src/modules/Gamma/info.json]: Oh Actually the gamma correction is the inverse of the actual gamma factor, so just to be extra clear I was thinking maybe we should mention that this is the gamma correction, inverse of the actual gamma factor. Thanks!

Comment (Mridul97) [reply src/modules/Gamma/info.json]: Oh okay, I will do that!

**Comment by Mridul97** [comment]
I have made the changes. Please have a look!

**Comment by jywarren** [comment]
Great :-) 👍 👍 👍 

Now that @Mridul97 knows the module format, they can help other people get their modules into the right format! One thing that can help is to have a "Saved reply" which links to the module docs. 

https://github.com/settings/replies

It could have a friendly message and link to: https://github.com/publiclab/image-sequencer/blob/master/CONTRIBUTING.md

Actually we could also have a pull request template linking to that as well! See some of the templates we can install here: 

https://github.com/publiclab/plots2/tree/master/.github/

**Pull request merged**